static/js/pill_typeahead.js

```diff
@@ -64,10 +64,17 @@ export function set_up(input, pills, opts) {
                 return typeahead_helper.render_stream(item);
             }
 
-            if (include_user_groups) {
-                return typeahead_helper.render_person_or_user_group(item);
+            if (include_user_groups && user_groups.is_user_group(item)) {
+                return typeahead_helper.render_user_group(item);
+                // if (include_users) {
+                //     return typeahead_helper.render_person_or_user_group(item);
+                // }
+                // return typeahead_helper.render_user_group(item);
```

Comment:
What's the story with this commented code?

Reply:
Oh, I've removed it now. Usually I do refactoring by commenting older pieces and then writing newer ones to ensure I am not changing the logic unknowingly.

Reply:
Sure, just it's good to do a self-review afterwards to catch these things.

(That said, I prefer to delete the code and see it via `git diff` if need be, since that leaves me with less cleanup to do at the end).